config/crds/cdap_v1alpha1_cdapmaster.yaml

```diff
@@ -235,6 +241,10 @@ spec:
             serviceAccountName:
               description: The service account for all the service pods.
               type: string
+            upgradeJobStartTimeNano:
+              description: Upgrade Job start time in nano, for internal use only
```

Comment:
How is it being used and who is using it? For internal states used by the controller, they should be stored in the status section. The spec section is for user to define the specification, which shouldn’t have any internal stuff